ksqldb-functional-tests/src/test/resources/test-runner/correct/from_docs/input.json

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think this is an accidental change

Reply:
Can you remove this change from the PR please?

Reply:
done